easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,26 @@
+name = 'bzip2'
+version = '1.0.8'
+
+homepage = 'https://sourceware.org/bzip2'
+
+description = """
+ bzip2 is a freely available, patent free, high-quality data compressor. It
+ typically compresses files to within 10% to 15% of the best available
+ techniques (the PPM family of statistical compressors), whilst being around
+ twice as fast at compression and six times faster at decompression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://sourceware.org/pub/bzip2/']
+sources = ['/apps/common/UES/easybuild/sources/b/bzip2/bzip2-1.0.8.tar.gz']
```

Comment:
Just for the record, absolute path is used here because there is currently an issue downloading this file from the tds node.